plugins/fabric8-cdn-support/pom.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2016-2018 Red Hat, Inc.
```

Comment:
2019 🎉

Reply:
I fear it's not supported by the license checker for now